### ISSUE TITLE
Unable to install with "brew install nakabonne/ali/ali"

### ISSUE THREAD

**Issue by rmgmediaval** [nakabonne/ali#53]
I noticed this when installing:
````
$ brew install nakabonne/ali/ali
==> Tapping nakabonne/ali
Cloning into '/usr/local/Homebrew/Library/Taps/nakabonne/homebrew-ali'...
remote: Enumerating objects: 39, done.
remote: Counting objects: 100% (39/39), done.
remote: Compressing objects: 100% (26/26), done.
remote: Total 39 (delta 13), reused 0 (delta 0), pack-reused 0
Receiving objects: 100% (39/39), 6.26 KiB | 6.26 MiB/s, done.
Resolving deltas: 100% (13/13), done.
Tapped 1 formula (26 files, 33.6KB).
Error: No available formula or cask with the name "nakabonne/ali/ali".
==> Searching for similarly named formulae...
This similarly named formula was found:
ali
To install it, run:
  brew install ali
~ ⌚ 9:02:48
$ brew install ali              
==> Installing ali from nakabonne/ali
==> Downloading https://github.com/nakabonne/ali/releases/download/v0.3.3/ali_0.3.3_darwin_amd64.tar.gz
==> Downloading from https://github-production-release-asset-2e65be.s3.amazonaws.com/294836885/d0365a80-0
######################################################################## 100.0%
🍺  /usr/local/Cellar/ali/0.3.3: 5 files, 6.2MB, built in 2 seconds
```
`

**Comment by nakabonne** [comment]
@rmgmediaval Hi, thank you for reporting! According to [Homebrew's documentation](https://docs.brew.sh/Taps#formula-with-duplicate-names), the way to use fully qualified names seems to be recommended. And in my local machine, it actually works at Homebrew v2.5.2:
```
❯ brew -v
Homebrew 2.5.2

❯ brew install nakabonne/ali/ali
==> Installing ali from nakabonne/ali
==> Downloading https://github.com/nakabonne/ali/releases/download/v0.3.3/ali_0.3.3_darwin_amd64.tar.gz
🍺  /usr/local/Cellar/ali/0.3.3: 5 files, 6.2MB, built in 2 seconds
```

Could you let me know what version do you use?

**Comment by rnevius** [comment]
This appears to be a problem with homebrew 2.5.3: https://github.com/ChargePoint/xcparse/issues/54

**Comment by rmgmediaval** [comment]
I have Homebrew v 2.5.3

**Comment by nakabonne** [comment]
@rnevius Thanks for the valuable information!

@rmgmediaval Can you try to update it and see if you can install it?

**Comment by rmgmediaval** [comment]
@nakabonne I think there is a small misunderstanding. I was able to install ali when I posted my initial comment (see above). 
I just thought I'd help by letting you know that I wasn't able to do it with qualified names. I simply wasn't aware it was because of my homebrew version. 

Furthermore, I updated my homebrew installation and tried installing the cask the way it's described in the README. It attempts to do it now.

`$ brew install nakabonne/ali/ali
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (nakabonne/ali).
==> Updated Formulae
Updated 1 formula.

==> Installing ali from nakabonne/ali
==> Downloading https://github.com/nakabonne/ali/releases/download/v0.3.4/ali_0.3.4_darwin_amd64.tar.gz
==> Downloading from https://github-production-release-asset-2e65be.s3.amazonaws.com/294836885/c5aacb00-0
######################################################################## 100.0%
Error: ali 0.3.3 is already installed
To upgrade to 0.3.4, run `brew upgrade nakabonne/ali/ali`.

`

**Comment by agoddard** [comment]
Tested this on 2.5.5, and it works fine now, the problem was introduced in homebrew 2.5.3 and fixed in 2.5.4 https://github.com/Homebrew/brew/pull/8875

**Comment by nakabonne** [comment]
@rmgmediaval @agoddard Thanks for giving it a try! Closing since we found out it was a Homebrew's bug that only occurred at v2.5.3.
